rita/src/rita_common/dashboard/mod.rs

```diff
@@ -69,10 +72,17 @@ impl Handler<GetOwnInfo> for Dashboard {
                         let balance = balance
                             .checked_div(&Int256::from(1_000_000_000i64))
                             .ok_or(OwnInfoError::RoundDownError(balance.clone()))?;
+ 
+                        let device = match SETTING.get_network().device.clone() {
```

Comment:
Shorter: `let device = SETTING.get_network().device.clone().unwrap_or(String::new())` (I think if `device` is a `String` already you don't need to call .to_string()). Also, wouldn't it make more sense from the frontend side to send "null" value when its missing?